cmp/leftRightChooser/LeftRightChooserFilter.js

```diff
@@ -16,12 +17,14 @@ import {escapeRegExp} from 'lodash';
  */
 @hoistComponent()
 class LeftRightChooserFilter extends Component {
-    @setter @observable value = '';
 
-    static defaultProps = {
-        fields: []
+    static propTypes = {
+        /** width of field, in pixels */
+        fields: PT.arrayOf(PT.string)
```

Comment:
Nope 😄 
But this is also a pure implementation class, so I don't know that we need the propTypes here.

Reply:
Actually, think Gil had in mind that this was going to be part of the API as an auxillary object.  Apps have to add it, and put it somewhere in the layout.

Reply:
Ah I see - in particular I see now that this field is not included in the LR chooser component itself as I thought it was. Let's see how it shakes out - I can understand how different uses of the component might/might not need/want the filter or a toolbar or otherplace to put it.

Reply:
No definitely not. Sorry will correct as I am going through.